### DISCUSSION TITLE
After Successfull Build, What to do next with quarkus, how to run it in local

### DISCUSSION THREAD

**Discussion by piyush-1234** [quarkusio/quarkus#29223] [Community]
i have successfully build quarkus locally now i want to start focusing on issues. 

first thing I want to run quarkus locally can anyone guide me or can drop some way to do it

**Comment by maxandersen** [comment]
'quarkus dev'?

**Reply by piyush-1234** [reply]
hi @maxandersen thanks for reply, can you help me in how to run quarkus, I have successfully built it locally, now I want to start working on it, need more clarity about flow and all,

**Reply by maxandersen** [reply]
If you have quarkus cli installed run Quarkus dev

Or if maven project run mvn quarkus:dev

You can follow https://quarkus.io/get-started/ to get step by step info to get started.

**Comment by piyush-1234** [comment]
yes
